### PR TITLE
80 add news to app & bugfix for fragments

### PR DESCRIPTION
**Pull request by Araknor99**
The news have been implemented and the bugfix for the fragments has been added.

### CODE REVIEW

File: app/src/main/res/layouts/courses/layout/item_news.xml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    <android.support.v7.widget.CardView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:id="@+id/card_view"
+        android:orientation="vertical">
+
+        <RelativeLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:paddingTop = "10dp"
```
Comment (Langleu) [inline app/src/main/res/layouts/courses/layout/item_news.xml]: whitespaces

Comment (Langleu) [reply app/src/main/res/layouts/courses/layout/item_news.xml]: same for paddingBottom and Right ' = "..."' --> '=".."'

File: app/src/main/java/org/schulcloud/mobile/ui/news/NewsPresenter.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                    @Override
+                    public void onError(Throwable e) {
+                        Timber.e(e,"Es gab einen Fehler beim Laden der News");
+                        getMvpView().showError();
+                    }
+
+                    @Override
+                    public void onNext(List<News> newses) {
+                        if(newses.isEmpty()){
+                            System.out.println(newses.size());
```
Comment (Langleu) [inline app/src/main/java/org/schulcloud/mobile/ui/news/NewsPresenter.java]: System.out.println?

Comment (Araknor99) [reply app/src/main/java/org/schulcloud/mobile/ui/news/NewsPresenter.java]: Oh, it seems like I forgot to remove that after debugging.

**Pull request merged**